### DISCUSSION TITLE
Huge spread offers.

### DISCUSSION THREAD

**Discussion by MwithM** [bisq-network/bisq#5391] [General]
What do you prefer? Empty orderbooks or orderbooks "filled" with completely out-of-market offers?
Knowing that every offer placed at Bisq consumes other people's resources to keep the P2P network, I prefer to see them empty. 
About how to solve this, I don't know what's the best and it won't be an easy task probably. There have been some [ideas](https://github.com/bisq-network/proposals/issues/268) though.
Just remember that if your offer is not likely to be taken, it's better to dehabilitate it at Portfolio - Open offers.

Related: https://www.reddit.com/r/bisq/comments/mj61d7/the_rates_sellers_charge_make_no_sense_to_me/

**Comment by pazza83** [comment]
One offer is at a 95% premium to sell LTC. How was the user able to make this offer? The application limits offers to 50% premium?

I prefer order books with out-of-market offers as hopefully it encourages traders to come in and make more competitive offers. This is what happens on all but the most illiquid markets. If volume increased on the LTC market prices would get tighter.

**Reply by ghost** [reply]
- LTC and other altcoins there's no use-case for trading them on Bisq with the friction of 4 BTC txn per trade priced in.  
- XMR actually has a use-case and therefore people need to exchange it for BTC, which explains its vibrant market on Bisq.  
- BSQ has a use-case too, so people are willing to pay the price of trading it.
